windrecorder/utils.py

```diff
@@ -609,3 +627,11 @@ def change_startup_shortcut(is_create=True):
             print("record: Shortcut already exists")
             os.remove(shortcut_path)
             print("record: Delete shortcut")
+
+
+def is_win11():
```

Comment:
Remove if this is not used

Reply:
I plan to reuse this func in the future for some different performances of win10 and win11, I still want to keep it for now